UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -183,6 +183,16 @@ <h2>Publisher metadata ecosystem</h2>
                         <p>
                             Finally from the "Metadata Processing" layer both the Display Techniques for EPUB and ONIX flow into the "Resulting Statements" layer "User Experience Guide for Displaying Accessibility Metadata"</p>
                     </details>
+</section>
+
+<section id="legal-considerations">
+<h2>Legal Considerations</h2>
+
+<p> in some jurisdictions  publishers may have an exemption from the provision of accessible publications, including the provision of accessibility metadata. Some examples of this are when the publisher claims a disproportionate burden  exemption or when it is a micro-enterprise (e.g., enterprises employing fewer than 10 people and with annual
```

Comment:
Small typo, start of sentence "In" should be capitalized.

Reply:
If you're citing the definition of a microenterprise, it should be "i.e.," not "e.g.,". Using "e.g." suggests there are other kinds of microenterprises beyond the kind listed.